src/pages/generate.tsx

```diff
@@ -537,8 +568,29 @@ export default function ClientIdentifierGenerator() {
                           </Grid>
                         </AccordionSummary>
                         <AccordionDetails sx={{ padding: 0 }}>
-                          <Grid container spacing={2}>
+                          <Grid container spacing={2} paddingTop={-1}>
                             <Grid item>
+                              <Grid
+                                container
+                                item
+                                direction="row"
+                                spacing={1}
+                                justifyContent="end"
+                              >
+                                <Grid item>
+                                  <Typography fontSize={12}>
+                                    Field name:
+                                  </Typography>
+                                </Grid>
+                                <Grid item>
+                                  <Typography
+                                    fontSize={12}
+                                    fontFamily="Courier Prime"
+                                  >
+                                    require_auth_time
```

Comment:
Would be best to wrap this up into a reusable component as you've the same markup in multiple places